Scripts/travis-build-test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -x -o pipefail
+
+echo "Building"
+
+rm -rf ${HOME}/Library/Developer/Xcode/DerivedData/*
+
+# -jobs -- specify the number of concurrent jobs
+# `sysctl -n hw.ncpu` -- fetch number of 'logical' cores in macOS machine
+xcodebuild -jobs `sysctl -n hw.ncpu` test -workspace Contentful.xcworkspace -scheme Contentful \
+  -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 6s,OS=9.3" \
```

Comment:
It's pretty common practice. Because this is running Swift 3, there is no need to run a matrix. Effectively, iOS simulators are the same—the difference being screen size. There are no snapshot tests here so the screen real-estate variance has no effective difference.